instrumentation/micrometer/micrometer-1.5/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/micrometer/v1_5/CounterTest.java

```diff
@@ -5,18 +5,78 @@
 
 package io.opentelemetry.javaagent.instrumentation.micrometer.v1_5;
 
-import io.opentelemetry.instrumentation.micrometer.v1_5.AbstractCounterTest;
+import static io.opentelemetry.sdk.testing.assertj.MetricAssertions.assertThat;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.attributeEntry;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Metrics;
 import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
 import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-class CounterTest extends AbstractCounterTest {
+class CounterTest {
```

Comment:
I had to change the instrumentation name, so I just inlined all the tests - the testing module will get removed after 1.13 anyway.